File: interview_prep/algorithm/java/src/test/java/hoa/can/code/SqrtTest.java
package hoa.can.code;

import hoa.can.code.ez.Sqrt;
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class SqrtTest {
    Sqrt sol = new Sqrt();

    @Test
    public void test    () {
        assertEquals(1, sol.sqrt(1));
        assertEquals(2, sol.sqrt(4));
        assertEquals(3, sol.sqrt(9));
    }
}

File: interview_prep/algorithm/Kotlin/src/test/kotlin/SqrtTest.kt
import hoa.can.code.Sqrt
import org.junit.jupiter.api.Test
import kotlin.test.DefaultAsserter.assertEquals

internal class SqrtTest {
    private val tc: Sqrt = Sqrt()

    @Test
    fun testSqrt() {
        assertEquals("1^1=1", 1,tc.sqrt(1))
        assertEquals("2^2=4", 2, tc.sqrt(4))
        assertEquals("3^3=9", 3,tc.sqrt(9))
    }
}